fbpic/lpa_utils/bunch.py

```diff
@@ -153,38 +154,59 @@ def add_elec_bunch_gaussian( sim, sig_r, sig_z, n_emit, gamma0,
         boosted-frame simulations.
         `z_injection_plane` is always given in the lab frame.
     """
-    # Get Gaussian particle distribution in x,y,z
-    x = sig_r * np.random.normal(0., 1., N)
-    y = sig_r * np.random.normal(0., 1., N)
-    z = zf + sig_z * np.random.normal(0., 1., N) # with offset in z
-    # Define sigma of ux and uy based on normalized emittance
-    sig_ur = (n_emit/sig_r)
-    # Get Gaussian distribution of transverse normalized momenta ux, uy
-    ux = sig_ur * np.random.normal(0., 1., N)
-    uy = sig_ur * np.random.normal(0., 1., N)
-    # Now we imprint an energy spread on the gammas of each particle
+    # Generate Gaussian gamma distribution of the beam
     if sig_gamma > 0.:
         gamma = np.random.normal(gamma0, sig_gamma, N)
     else:
-        # Or set it to zero
+        # Zero energy spread beam
         gamma = np.full(N, gamma0)
         if sig_gamma < 0.:
             print("Warning: Negative energy spread sig_gamma detected."
```

Comment:
I know this was not modified by your PR, but could you use `warnings.warn` here instead of print?
(see other comment below)